src/main/resources/view/PersonListCard.fxml

```diff
@@ -28,7 +28,6 @@
         <Label fx:id="name" text="\$first" styleClass="cell_big_label" />
       </HBox>
       <FlowPane fx:id="tags" />
-      <Label fx:id="phone" styleClass="cell_small_label" text="\$phone" />
```

Comment:
Good job for removing this so it wont show up in the GUI

Reply:
Thank you Willy.